src/schemas/user.schema.js

```diff
@@ -2,15 +2,57 @@ var mongoose = require('mongoose');
 var Schema = mongoose.Schema;
 
 var userSchema = new Schema({
-  name: {
+  Id: {
     type: String,
     required: true
   },
+  Fname: {
```

Comment:
I prefer to use camelCase and a complete property name. this should be `firstName` same goes with the other properties

Reply:
Ok copy